markdown/preamble/license.md

```diff
@@ -3,4 +3,4 @@ License
 
 SMuFL is free to use, and is licensed under the terms of the W3C Community Contributor License Agreement (CLA) Deed:
 
-> Copyright © 2017 the Contributors to the Standard Music Font Layout Specification, published by the W3C Music Notation Community Group under the [W3C Community Contributor License Agreement (CLA)](https://www.w3.org/community/about/agreements/cla/). A [human-readable summary](http://www.w3.org/community/about/agreements/cla-deed/) is available.
+> Copyright © 2019 the Contributors to the Standard Music Font Layout Specification, published by the W3C Music Notation Community Group under the [W3C Community Final Specification Agreement (FSA)](https://www.w3.org/community/about/agreements/fsa/). A [human-readable summary](https://www.w3.org/community/about/agreements/fsa-deed/) is available.
```

Comment:
As on the HTML page, the last component of the first URL should be final rather than fsa. I assume changing it here and rebuilding will change it in the HTML.